expressServer/scripts/encryptUser.js

```diff
@@ -0,0 +1,65 @@
+const mongoose = require('mongoose');
+const dotenv = require('dotenv');
+const crypto = require('crypto');
+const { create } = require('domain');
+const prompt = require('prompt-sync')();
+
+dotenv.config();
+
+mongoose.connect(process.env.MONGODB_URI, {})
+  .then(() => console.log('Connected to MongoDB'))
+  .catch(err => console.error('Could not connect to MongoDB', err));
+
+const userSchema = new mongoose.Schema({
+  username: { type: String, required: true, unique: true },
+  hash: { type: String, required: true },
+  salt: { type: String, required: true }
+});
+
+const User = mongoose.model('User', userSchema);
+
+function hashPassword(password, salt) {
+  return crypto.createHash('sha3-512').update(password + salt).digest('hex');
+}
+
+function promptForUsernameAndPassword() {
```

Comment:
Get parameters from command line and not from the prompt, the execution will be `node encryptUser username password`, you should be able to retrieve the values without human interaction